src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="get"
+      class="form"
+      data-qa="big"
+    >
+      <label class="search_label search_label_big">
```

Comment:
a little fix. It applies to BEM methodology
The same as for the next form label
```suggestion
      <label class="search__label search_label--big">
```